arviz/data/base.py

```diff
@@ -3,9 +3,9 @@
 import datetime
 import warnings
 
-import numpy as np
 import pkg_resources
 import xarray as xr
+from ..utils import two_de, arange
```

Comment:
Reading through this code I have a slight preference that arange should be under `utils` namescope
so this like would be `from .. import utils` and then subsequent calls would be `utils.arange`

The reason for this is that this way its really obvious its not the `np.arange` method but another one of our own design. It does add characters but I think it removes a mental burden that makes it worthwhile

thoughts?

Reply:
Sure